ledger/sandbox/src/main/scala/com/digitalasset/platform/store/ActiveLedgerStateManager.scala

```diff
@@ -206,7 +199,8 @@ class ActiveLedgerStateManager[ALS <: ActiveLedgerState[ALS]](initialState: => A
                       )
                     } else {
                       ats.copy(
-                        errs = errs + InvalidLookup(gk, nlkup.result, currentResult)
+                        errs = errs + Disputed(
+                          s"Contract key lookup with different results: expected [${nlkup.result}], actual[${currentResult}]")
```

Comment:
Is this an improvement related to the fix? I lack bg. on the previous error reporting but can you elaborate here?

Reply:
See my [previous comment](https://github.com/digital-asset/daml/pull/5608#discussion_r411343492).